docs/dns/rrsets.rst

```diff
@@ -213,6 +213,20 @@ https://www.huque.com/bin/gen_tlsa.  We are planning to provide a tool that is
 connected directly to our API in the future.  For full detail on how ``TLSA``
 records work, please refer to RFC 6698.
 
+The ``TLSA`` sha256 hash can also be generated with that command::
+
+    openssl x509 -in /etc/letsencrypt/live/${name}/fullchain.pem -noout -pubkey \
```

Comment:
The path to the `fullchain.pem` file depends on the OS and configuration. On OpenBSD they tend to live in `/etc/ssl/acme/…` for example.

And the variable `$name` is not defined anywhere.

Maybe make this more clear by using an obvious placeholder like `<path-to-fullchain.pem>`.